aws_lambda_powertools/utilities/parameters/appconfig.py

```diff
@@ -68,14 +70,19 @@ def __init__(
         application: Optional[str] = None,
         config: Optional[Config] = None,
         boto3_session: Optional[boto3.session.Session] = None,
+        boto3_client: Optional[boto3.client] = None,
```

Comment:
Same as the SSM comment: https://youtype.github.io/boto3_stubs_docs/mypy_boto3_appconfig/client/

Reply:
fixed